src/axom/inlet/VariantKey.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/inlet/VariantKey.hpp"
+
+namespace axom
+{
+namespace inlet
+{
+VariantKey& VariantKey::operator=(const VariantKey& other)
+{
+  copyFrom(other);
+  return *this;
+}
+VariantKey& VariantKey::operator=(VariantKey&& other)
```

Comment:
You don't need to fix this... but it is an obvious oversight that no newline is required here...